docs/semgrep-app/sso.md

```diff
@@ -61,4 +61,21 @@ To set up SSO:
 
 If you encounter issues during the setup process, reach out to [support@r2c.dev](mailto:support@r2c.dev) for assistance.
 
-<MoreHelp />
+### Setting up SAML SSO with Azure Active Directory
+
+Set up Semgrep as an Enterprise App with SAML SSO using Azure's Active Directory service by following these steps:
+
+1. Enable **Enterprise App** in **Azure Active Directory** with SAML SSO selected. Follow the instructions in [Enable single sign-on for an enterprise application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal-setup-sso).
+2. Go to Semgrep App [Settings](https://semgrep.dev/orgs/-/settings/sso) page. TODO Under **Basic Configuration**, you'll need to enter the `Entity ID` and the `Reply URL`. These are provided in the Semgrep App on the left-side **Settings** tab, under the **SSO** tab on the top, as `Audience URL` and `SSO URL`, respectively. Make sure you switch the provider type to **SAML2** in the SSO form in the Semgrep App.
+    !["Entity ID" and "Reply URL" fields in Semgrep App](/img/azure-sso-fields-2.png ""Entity ID" and "Reploy URL" fields from within the App")
+3. TODO You need to add two attribute statements called `name` and `email`, making sure to set the `email` attribute to `user.mail`. For help regarding **Attributes and Claims**, refer to [Customize claims issued in the SAML token for enterprise applications](https://learn.microsoft.com/en-us/azure/active-directory/develop/active-directory-saml-claims-customization) documentation.
+4. In the Azure Single sign-on page, find the `Login URL`, `Azure AD Identifier`, and `Certificate (Base64)`. Pate them to your [Settings](https://semgrep.dev/orgs/-/settings/sso) page of SAML2 SSO as `IdP SSO URL` (`Login URL`), `IdP Issuer ID` (`Azure AD Identifier`), and `X.509 Certificate` (`Certificate (Base64)`). 
+    ![Setting additional SSO form fields](/img/azure-sso-app-fields.png "Setting additional SSO form fields")
+5. TODO At the top of the form, fill in your email domain and a display name for the SSO integration.
```

Comment:
This seems to be prefilled already (I checked in two organizations).

Reply:
FYI, on my end it was not prefilled @adamkvitek . Probably depends on the version of Semgrep App when the org was added.
<img width="581" alt="2022-12-09 20_02" src="https://user-images.githubusercontent.com/20766840/206698492-d714d5c7-ee24-443f-a6ae-41b2a331117b.png">